app/templates/relay/_hubs.html

```diff
@@ -1,17 +1,15 @@
 <table>
-  {% for h in hubs %}
-  {% if loop.first %}
   <tr>
     <th>Time</th>
-    <th>{% if h.hub_id is defined %}XBee{% endif %}</th>
-    <th>{% if h.pi_id is defined %}Pi ID{% endif %}</th>
+    <th>Hub ID</th>
+    <th>Pi ID</th>
```

Comment:
It seems that you would want the header even if the first row of the cell didn't have a value for it because subsequent rows might. Right?